quickstart/101-virtual-network-manager-create-management-group-scope/providers.tf

```diff
@@ -0,0 +1,16 @@
+terraform {
+  required_version = ">=1.0"
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = ">= 3.56.0"
```

Comment:
@mbender-ms What are the constraints you're looking for here?

Reply:
I would recommend `">= 3.56.0, < 4.0"` here since `4.0` might contain numerous breaking changes.

Reply:
@lonegunmanb Wouldn't the pessimistic constraint operator accomplish the same thing. I believe that the following means to increment the minor version but never the major version. In other words, all versions before 4.0.

`~> 3.56`

Reply:
@TomArcherMsft I re-using what was in the current template. I did not re-create this from scratch based off of the templates recommended in the updated guide.

 As I recall, those values were specified to me by someone on the terraform team during the creatin process of the original. I did

Reply:
Added `~> 3.56.0, > 4.0`